soc/v2/xilinx/zynqmp/Kconfig.soc

```diff
@@ -2,6 +2,9 @@
 # Copyright (c) 2019 Stephanos Ioannidis <root@stephanos.io>
 # SPDX-License-Identifier: Apache-2.0
 
+config SOC_XILINX_ZYNQMP
+	bool
+
 config SOC_XILINX_ZYNQMP_RPU
 	bool "Xilinx ZynqMP RPU"
 	select ARM
```

Comment:
`select ARM` must go to the `xilinx/zynqmp/Kconfig` file.

Reply:
Fixed